flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js

```diff
@@ -135,19 +163,27 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
     refreshADIRS() {
         if (this.tatText != null && this.satText != null) {
             if (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_STATE", "Enum") != 2) {
+                this.areAdirsAligned = false;
                 this.tatText.textContent = "XX";
                 this.tatText.classList.add("Warning");
                 this.tatText.classList.remove("Value");
                 this.satText.textContent = "XX";
                 this.satText.classList.add("Warning");
                 this.satText.classList.remove("Value");
             } else {
+                this.areAdirsAligned = true;
                 this.satText.classList.add("Value");
                 this.satText.classList.remove("Warning");
                 this.tatText.classList.add("Value");
                 this.tatText.classList.remove("Warning");
+                this.isaText.classList.add("Value");
+                this.isaText.classList.remove("Warning");
             }
         }
     }
+    getISA() {
+        // EIS2 S13-2 ref
+        return this.currentSAT + (1013 * 2 / 1000) - 15;
```

Comment:
(1013 * 2 / 1000) - 15 could be simplified to a const 

```suggestion
        return this.currentSAT -12,974;
```

wondering why the Isa calculation is that different from the fmgc...

![image](https://user-images.githubusercontent.com/1857322/121314153-5f8d9380-c907-11eb-894d-cd8b93a6b6af.png)